src/status_im/chat/models/message.cljs

```diff
@@ -31,8 +31,8 @@
 (defn- prepare-message
   [{:keys [content] :as message} chat-id current-chat?]
   ;; TODO janherich: enable the animations again once we can do them more efficiently
-  (cond-> (assoc message :appearing? true)
-    (not current-chat?) (assoc :appearing? false)
+  (cond-> (assoc message :appearing? true :animate? true)
+    (not current-chat?) (assoc :appearing? false :animate? false)
```

Comment:
Minor point, `:appearing?` and `:animate?` are actually the same things, we should use one or another. But that's a minor point can be done later.

Reply:
I considered to utilize the flag `:appearing?`, however in the animation case, we need to switch the flag once the animation is done, which seems to be not the case for `:appearing?` today. 
Actually, I am not quite sure about the original purpose of `:appearing?`, so I decided to get around that flag by introducing a new one.

Reply:
`:appearing?` was used for exactly the same purpose as `:animate?` is now.
Previously, we did have an message appearing animation, but we decided to remove it (but we left the flag) because of the performance problems.

Reply:
this animation should be much less performance penalizing, because of the native driver, but worth checking that on an old Android anyways

Reply:
@mandrigin That's true, but still, in situation where you open a busy chat, and you get many messages in span of very short time (offline inboxing), it could be problematic when all of them are animated -> previously we had a problem that the application (already far stretched by processing the messages) became totally unresponsive for extended periods of time (15s+), so that's something we definitely have to check. 
In case it's still a problem we need to implement more advanced time buffering, where only the last received message in span of some time (to be tuned) buffer is actually animated.